usr/lib/common/pkey_utils.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * COPYRIGHT (c) International Business Machines Corp. 2020
+ *
+ * This program is provided under the terms of the Common Public License,
+ * version 1.0 (CPL-1.0). Any use, reproduction or distribution for this
+ * software constitutes recipient's acceptance of CPL-1.0 terms which can be
+ * found in the file LICENSE file or at
+ * https://opensource.org/licenses/cpl1.0.php
+ */
+
+#ifndef PKEY_UTILS_H
+#define PKEY_UTILS_H
+
+#include "pkcs11types.h"
+#include "ec_curves.h"
+
+/**
+ * the pkey kernel module
+ */
+#define PKEYDEVICE "/dev/pkey"
+
+/**
+ * Modifier bit for CPACF instructions.
+ */
+#define CPACF_ENCRYPT         0x00
+#define CPACF_DECRYPT         0x80
+
+/**
+ * function codes for the KM/KMC/KMAC instruction.
+ */
+#define Encrypted_AES_128     0x1a
```

Comment:
Can those be all UPPERCASE, like other defines?

Reply:
Sure, done.